cmake/Utils.cmake

```diff
@@ -0,0 +1,40 @@
+macro(normalize_path in_path)

```

Comment:
You should use `include_guard()`. Note that it's cmake 3.10, though.

Reply:
see latest file